lib/services/book.ts

```diff
@@ -86,5 +86,8 @@ export async function getPostsForBook(
     where: {
       book: { isbn },
     },
+    orderBy: {
```

Comment:
We also need this in `lib/services/user.ts` in `getUserWithPosts` for the account page, but I am okay with just this change for now.

Reply:
Ah I knew I was missing something, thanks

Reply:
See https://github.com/calebmech/Book-Bazar/commit/46120175267f47c502492b12110d3bebf6da44f5​​​​​​​